packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -331,6 +335,14 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
         return true;
     }
 
+    /** @internal */
+    public get _rawBoundingInfo(): Nullable<BoundingInfo> {

```

Comment:
Since we're already having to add the getters and setters, we could make the property public?

Reply:
Should be fixed. Let me know if I missed your point.